plugins/aws/resoto_plugin_aws/collector.py

```diff
@@ -22,6 +22,7 @@
     s3,
     service_quotas,
     kinesis,
+    redshift,
```

Comment:
looks like you need to update your base branch 💡 - also I think this list is sorted alphabetically? Same for the resources in line 48

Reply:
I'm on the latest master, not sure what is newer :) 

Regarding the list, I think the order does not matter.